api/src/main/java/org/jenkinsci/plugins/workflow/flow/StashManager.java

```diff
@@ -73,9 +64,11 @@
      * @param workspace a directory to use as a base
      * @param includes a set of Ant-style file includes, separated by commas; null/blank is allowed as a synonym for {@code **} (i.e., everything)
      * @param excludes an optional set of Ant-style file excludes
+     * @param useDefaultExcludes whether to use Ant default excludes
      */
     @SuppressFBWarnings(value="RV_RETURN_VALUE_IGNORED_BAD_PRACTICE", justification="fine if mkdirs returns false")
-    public static void stash(@Nonnull Run<?,?> build, @Nonnull String name, @Nonnull FilePath workspace, @Nonnull TaskListener listener, @CheckForNull String includes, @CheckForNull String excludes) throws IOException, InterruptedException {
+    public static void stash(@Nonnull Run<?,?> build, @Nonnull String name, @Nonnull FilePath workspace, @Nonnull TaskListener listener,
+                             @CheckForNull String includes, @CheckForNull String excludes, boolean useDefaultExcludes) throws IOException, InterruptedException {
```

Comment:
Yeah…in cases like these, most people will update all the WF plugins simultaneously, but Plugin Manager does not enforce this, so I like to err on the side of caution. If and when we have a system for core & plugins to declare that particular downstream plugins are known to be broken by an update and must be updated in tandem, we can start to delete the deprecated detritus.